Ukoly/DU_4/Caesar.py

```diff
@@ -0,0 +1,38 @@
+key = input('Key: ')
+while key.isnumeric() is False or (int(key) < 0):
+    print('Key has to be positive number.')
+    key = input('Key: ')
+key = int(key) % 26
+
+convert = list()
```

Comment:
Krajšie je možno použiť `[]` ale to je maličkosť 😄.

Reply:
Omg 😄 na jednom místě is False opravím a na druhém ne. No, snad už si to zapamatuju jednou pro vždy.